schema/spec.md

```diff
@@ -270,9 +276,12 @@ core xRegistry Resource
   specification defines a set of common [schema format names](#schema-formats)
   that MUST be used for the given formats, but applications MAY define
   extensions for other formats on their own.
+
+  For many schema registry use cases this attribute might be very important for
```

Comment:
```suggestion
  For many schema registry use cases this attribute is important for
```

There's already hedging in "For many schema registry use cases", so I don't think we need the "might be" (and "very" doesn't feel helpful to me - although that's a personal preference of course).

Reply:
done